<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="data_dir"          default="/mnt/md0/gear_data"/>
  <arg name="image_extn"        default=".jpg"/>
  <arg name="frame_rate"        default="30"/>
  <arg name="image_prefix"      default="im"/>
 
  <!-- Synchronized logger node -->
  <node name="synchronized_logger" pkg="gear_image_handler" type="synchronized_image_logger.py"  output="screen">
    <param name="data_dir"      value="$(arg data_dir)"/>
    <param name="image_extn"    value="$(arg image_extn)"/>
    <param name="frame_rate"    value="$(arg frame_rate)"/>
    <param name="image_prefix"  value="$(arg image_prefix)"/>
  </node>

</launch>